src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParser.cs

```diff
@@ -7278,7 +7278,7 @@ internal Task TdsExecuteSQLBatch(string text, int timeout, SqlNotificationReques
                 }
                 else
                 {
-                    throw e;
+                    throw;
```

Comment:
Wouldn't this be sufficient to surface the complete stack trace which shows where the exception came from?

Reply:
In this case, yes. But I'm not sure if the same issue with DbDataReader.GetSchemaTable could pop up elsewhere

Reply:
From the risk perspective, I would say let's fix this part for now and if it crops up again and is a major pain point, we do a code scan to figure out where else do we swallow these exceptions.

Reply:
Removed the other changes